src/Eloquent/Builder.php

```diff
@@ -325,6 +325,9 @@ protected function addUpdatedAtColumn(array $values)
         }
 
         $column = $this->model->getUpdatedAtColumn();
+        if (array_key_exists('$set', $values) && array_key_exists($column, $values['$set'])) {
+            return $values;
+        }
```

Comment:
We always have a blank line after closing brace. 
Please run `vendor/bon/phpcbf`